build_container.sh

```diff
@@ -74,7 +74,7 @@ rustup component add llvm-tools-preview  # needed for coverage
 # Install other rust targets.
 # Skip on `riscv64` for now
 if [ "$ARCH" != "riscv64" ]; then
-    rustup target add $ARCH-unknown-linux-musl $ARCH-unknown-none
+    rustup target add $ARCH-unknown-linux-musl $ARCH-unknown-none aarch64-apple-darwin x86_64-pc-windows-gnu
```

Comment:
why we are installing them for every "$ARCH" except for riscv64?

Reply:
oops, yes, that's wrong, the intention was to only install them on _one_ arch